MaterialDesignThemes.Wpf/AdornerAssist.cs

```diff
@@ -1,28 +1,44 @@
+using System;
 using System.Linq;
 using System.Windows;
 using System.Windows.Documents;
 
 namespace MaterialDesignThemes.Wpf
 {
     /// <summary>
-    /// this is a simple utility to add and remove a single adorner to an element since there is no built-in way to do that in xaml.
-    /// <a href="https://docs.microsoft.com/en-us/dotnet/framework/wpf/controls/adorners-overview">see here</a>
+    /// This is a simple utility to add and remove a single adorner to an element
+    /// since there is no built-in way to do that in xaml.
+    /// <a href="https://docs.microsoft.com/en-us/dotnet/framework/wpf/controls/adorners-overview">See here</a>
     /// </summary>
-    internal static class AdornerAssist
+    public static class AdornerAssist
     {
-        public static void AddAdorner<T>(this UIElement element, T adorner) where T : Adorner
+        public static void AddAdornerOfType<T>(this UIElement element, T adorner) where T : Adorner
```

Comment:
Rather than adding "OfType" to the method name, can we rename the generic parameter from `T` to `TAdorner` instead?

Reply:
I can agree with that. The OfType in this case is redundant information.
Renaming the generic parameter is sufficient.